src/main/java/demo/test_logger/NewLogger.java

```diff
@@ -0,0 +1,26 @@
+package demo.test_logger;
+
+import ch.qos.logback.classic.Level;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class NewLogger {
+    public static void main(String[] args) {
+
+        ch.qos.logback.classic.Logger parentLogger =
```

Comment:
why you use this class "ch.qos.logback.classic" ? if you use slf4j - then try not specify classes which are used behind the slf4j facade